scripts/target.rb

```diff
@@ -8,7 +8,9 @@ class Target
       'openstack' => ImagefactoryMetadata.new('openstack-kvm', nil, 'qc2'),
       'hyperv'    => ImagefactoryMetadata.new('hyperv', nil, 'vhd'),
       'azure'     => ImagefactoryMetadata.new('hyperv', nil, 'vhd'),
-      'vagrant'   => ImagefactoryMetadata.new('vsphere', 'vagrant-virtualbox', 'box')
+      'vagrant'   => ImagefactoryMetadata.new('vsphere', 'vagrant-virtualbox', 'box'),
+      'libvirt'   => ImagefactoryMetadata.new('openstack-kvm', nil, 'qc2'),
```

Comment:
@Fryguy the target name drives certain conditional statements in the kickstarts. So while the definition looks identical, it will result in a different appliance.